go/pkg/bertyprotocol/store_message.go

```diff
@@ -26,10 +27,13 @@ const groupMessageStoreType = "berty_group_messages"
 type messageStore struct {
 	basestore.BaseStore
 
-	devKS  DeviceKeystore
-	mks    *MessageKeystore
-	g      *bertytypes.Group
-	logger *zap.Logger
+	devKS     DeviceKeystore
+	mks       *messageKeystore
+	g         *bertytypes.Group
+	logger    *zap.Logger
+	cache     map[string][]ipfslog.Entry
```

Comment:
I think it would be wiser to store the messages in a group scoped circular cache with a fixed size to protect against attack where someone could fill the RAM of all the group members' devices by sending a lot of messages without ever sending the secret.

Reply:
It should not be forgotten that by closing/reopening the group, the problem of messages arrived before the secrets disappears since the replay is done naturally.
We can therefore say that a circular cache of 1024 (for example) messages is sufficient in the sense that if in a highly unlikely case it happens that 1024 messages are received before the corresponding secrets, the problem will be minimal for the user (open/close the group -> problem solved).

Reply:
:+1: for the circular cache.